.gitignore

```diff
@@ -6,6 +6,10 @@ __pycache__/
 # C extensions
 *.so
 
+kafka-venv/*
```

Comment:
in the documentation, you tell us to create a virtual container called my-venv, but here you include kafka-venv, i'd recommend to update the documentation and pull request to tell us to create kafka-venv instead of my-venv